### PR TITLE
DOC-249 Shopify Plugin

### PR DESCRIPTION
**Pull request by markzegarelli**

### CODE REVIEW

File: content/collections/data/en/amplitude-shopify-plugin.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+blueprint: data
+title: 'Amplitude Shopify Plugin'
+landing: false
+exclude_from_sitemap: false
+updated_by: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
+updated_at: 1729791421
+---
+The [Amplitude Shopify Plugin](https://apps.shopify.com/amplitude) enables you to instrument your Shopify store with an Amplitude features including [Autocapture](/docs/data/autocapture) and [Session Replay](/docs/session-replay).
+
+The Shopify Plugin plugin installs a version of the [Browser SDK](/docs/sdks/analytics/browser/browser-sdk-2), and adds the script before the `</head>` tag on each of your site's pages.
```
Comment (SpencerFleury) [inline content/collections/data/en/amplitude-shopify-plugin.md:12]:
```suggestion
The Shopify plugin installs a version of the [Browser SDK](/docs/sdks/analytics/browser/browser-sdk-2), and adds the script before the `</head>` tag on each of your site's pages.
```

File: content/collections/data/en/amplitude-shopify-plugin.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  - `[Amplitude] Type`
+  - `[Amplitude] Variant Title`
+  - `[Amplitude] Title`
+  - `[Amplitude] Vendor`
+
+## Install the plugin
+
+To add the plugin to your Shopify store:
+
+1. Log in to your Shopify account, and find the plugin in the [Shopify App Store](https://apps.shopify.com/amplitude). Click *Install*.
```
Comment (SpencerFleury) [inline content/collections/data/en/amplitude-shopify-plugin.md:38]:
```suggestion
1. Log in to your Shopify account and find the plugin in the [Shopify App Store](https://apps.shopify.com/amplitude). Click *Install*.
```

File: content/collections/data/en/amplitude-shopify-plugin.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+2. Confirm the required plugin permissions, and click *Install*.
```
Comment (SpencerFleury) [inline content/collections/data/en/amplitude-shopify-plugin.md:39]:
```suggestion
2. Confirm the required plugin permissions and click *Install*.
```

File: content/collections/data/en/amplitude-shopify-plugin.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+3. Locate your project's API key, and add it to the Amplitude Settings page in Shopify. Click *Connect*.
```
Comment (SpencerFleury) [inline content/collections/data/en/amplitude-shopify-plugin.md:40]:
```suggestion
3. Locate your project's API key and add it to the Amplitude Settings page in Shopify. Click *Connect*.
```

File: content/collections/data/en/amplitude-shopify-plugin.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+    {{partial:admonition type="note" heading="Ad blocking software"}}
+    Disable any enabled ad blocking software in your browser to ensure the plugin setup step can communicate with Amplitude.
```
Comment (SpencerFleury) [inline content/collections/data/en/amplitude-shopify-plugin.md:43]:
```suggestion
    If any ad blocking software is running in your browser, disable it to ensure the plugin setup step can communicate with Amplitude.
```

File: content/collections/data/en/amplitude-shopify-plugin.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    {{/partial:admonition}}
+
+4. A "Connection successful" message indicates the plugin is enabled and connected with Amplitude.
```
Comment (SpencerFleury) [inline content/collections/data/en/amplitude-shopify-plugin.md:46]:
```suggestion
A "Connection successful" message indicates the plugin is enabled and connected with Amplitude.
```

Comment (SpencerFleury) [reply content/collections/data/en/amplitude-shopify-plugin.md:46]: This isn't a step—the user doesn't do anything here—so it doesn't get a number.

**Review by SpencerFleury — CHANGES_REQUESTED** [review]

File: content/collections/data/en/amplitude-shopify-plugin.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+id: d7658703-c1c3-4cc4-a5e3-3b5674e41405
+blueprint: data
+title: 'Amplitude Shopify Plugin'
+landing: false
+exclude_from_sitemap: false
+updated_by: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
+updated_at: 1729791421
+---
+The [Amplitude Shopify Plugin](https://apps.shopify.com/amplitude) enables you to instrument your Shopify store with an Amplitude features including [Autocapture](/docs/data/autocapture) and [Session Replay](/docs/session-replay).
+
```
Comment (katiegeer) [inline content/collections/data/en/amplitude-shopify-plugin.md:11]: ### About Shopify
[Shopify](https://www.shopify.com/) is an all-in-one commerce platform that allows businesses of any size to create, customize, and manage online stores with ease. It offers tools for product listings, payments, shipping, and customer engagement, streamlining the selling process online, across social media, and in person. With built-in marketing features, analytics, and integrations with third-party apps, Shopify helps business owners track performance and drive sales.

File: content/collections/data/en/amplitude-shopify-plugin.md (hunk 1 above)
Comment (katiegeer) [inline content/collections/data/en/amplitude-shopify-plugin.md]: Amplitude also sends all [Default Events](https://amplitude.com/docs/sdks/analytics/browser/migrate-from-marketing-analytics-to-browser-sdk-2-0#default-event-tracking).

File: content/collections/data/en/amplitude-shopify-plugin.md (hunk 7 above)
Comment (katiegeer) [reply content/collections/data/en/amplitude-shopify-plugin.md:11]: ### Considerations

- **Performance**: The packages we utilize on the frontend is roughly 167kb. To give a rough idea of the performance impact I generated 6 different Lighthouse Performance reports. 3 with the Amplitude packages, and 3 without. Without the Amplitude packages, the average performance score was 98. With the packages, the average Performance score was 96.
- **Session replay may not display as expected**. Shopify’s UI heavily relies on animations, specifically the fade-in animation. The animation responsible for changing the opacity of the UI from 0 to 1 is not consistently picked up by session replay, which is why elements appear to be missing.

File: content/collections/data/en/amplitude-shopify-plugin.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+updated_at: 1729791421
+---
+
+[Shopify](https://www.shopify.com/) is an all-in-one commerce platform that allows businesses of any size to create, customize, and manage online stores with ease. It offers tools for product listings, payments, shipping, and customer engagement, streamlining the selling process online, across social media, and in person. With built-in marketing features, analytics, and integrations with third-party apps, Shopify helps business owners track performance and drive sales.
+
+The [Amplitude Shopify Plugin](https://apps.shopify.com/amplitude) enables you to instrument your Shopify store with an Amplitude features including [Autocapture](/docs/data/autocapture) and [Session Replay](/docs/session-replay).
+
+## Considerations
+
+* Performance: The packages Amplitude provides are roughly 167kb total size. In testing, Shopify stores with Amplitude installed registered an average lighthouse score of 96, compared to an average score of 98 without Amplitude.
```
Comment (zacbennett) [inline content/collections/data/en/amplitude-shopify-plugin.md:17]:
```suggestion
* Performance: The packages Amplitude provides are roughly 167kb total size, but will vary depending on which products you utilize. In testing, Shopify stores with Amplitude installed registered an average lighthouse score of 96, compared to an average score of 98 without Amplitude.
```

File: content/collections/data/en/amplitude-shopify-plugin.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+* Session replay may not display as expected. Shopify’s UI relies on animations, specifically the fade-in animation. Session Replay doesn't capture the animation consistently, which may cause elements to appear missing.
```
Comment (zacbennett) [inline content/collections/data/en/amplitude-shopify-plugin.md:19]: I think this is okay to take out. This ended up being an issue with my test store, and we haven't received reports from customers that they are running into this

File: content/collections/data/en/amplitude-shopify-plugin.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+  - `[Amplitude] Title`
+  - `[Amplitude] Vendor`
+
+## Install the plugin
+
+To add the plugin to your Shopify store:
+
+1. Log in to your Shopify account and find the plugin in the [Shopify App Store](https://apps.shopify.com/amplitude). Click *Install*.
+2. Confirm the required plugin permissions and click *Install*.
+3. Locate your project's API key and add it to the Amplitude Settings page in Shopify. Click *Connect*.
```
Comment (zacbennett) [inline content/collections/data/en/amplitude-shopify-plugin.md:52]: We also just added a step where a user needs to indicate where they want to store their data. 
image

**Comment by zacbennett** [comment]
There's a lot more info we can add here, specifically with regards to what events we sent and their properties. Take a look at this FAQ doc https://amplitude.atlassian.net/wiki/spaces/GROWTH/pages/2578874543/Shopify+App+FAQ

**Comment by markzegarelli** [comment]
> There's a lot more info we can add here, specifically with regards to what events we sent and their properties. Take a look at this FAQ doc https://amplitude.atlassian.net/wiki/spaces/GROWTH/pages/2578874543/Shopify+App+FAQ

Thanks @zacbennett! I updated the [article](https://amplitude-docs-git-doc-249-amplitude-inc.vercel.app/docs/data/amplitude-shopify-plugin) with the properties and events listed in the FAQ.

As far as Session Replay and Web Experiment, I'll mainly link out to those sections, but is there anything you think would be good to capture in this article? For example, I'll mention that for Session Replay, it needs a non-zero sample rate.

**Comment by zacbennett** [comment]
> > There's a lot more info we can add here, specifically with regards to what events we sent and their properties. Take a look at this FAQ doc https://amplitude.atlassian.net/wiki/spaces/GROWTH/pages/2578874543/Shopify+App+FAQ
> 
> Thanks @zacbennett! I updated the [article](https://amplitude-docs-git-doc-249-amplitude-inc.vercel.app/docs/data/amplitude-shopify-plugin) with the properties and events listed in the FAQ.
> 
> As far as Session Replay and Web Experiment, I'll mainly link out to those sections, but is there anything you think would be good to capture in this article? For example, I'll mention that for Session Replay, it needs a non-zero sample rate.

Awesome. With regards to the Web Experiment, there is a minor flicker when implementing web experiment. There is a manual work around this is mentioned [here](https://amplitude-docs-i2yd1y5ep-amplitude-inc.vercel.app/docs/web-experiment/implementation#shopify). Think it might be helpful to link here.

**Comment by zacbennett** [comment]
Two common questions that I've gotten which I think are worth including here are:
1. What is the difference between Order Created and Checkout Created?
2. As well as, how is user id determined?

Both of these are answered in the FAQ doc, and I think would be worthwhile to add here. Otherwise everything LGTM!

**Review by zacbennett — APPROVED** [review]

**Pull request merged**